apps/remote_control/test/lexical/remote_control/build/parse_error_test.exs

```diff
@@ -0,0 +1,183 @@
+defmodule Lexical.RemoteControl.Build.ParseErrorTest do
```

Comment:
Fo the entire case, don't assert against tuples of numbers. Those are super hard to reason about. We have RangeSupport that highlights ranges. Use that instead